_posts/2024-12-18-cockpit-331.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Cockpit 331
+author: jelle
+date: '2024-12-18'
+tags: cockpit, machines, files
+slug: cockpit-331
+category: release
+summary: 'Prevent search engine indexing, support sharing host ssh-agent, downloading of unsupported operating systems and file upload improvements'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 331, cockpit-machines 326, and cockpit-files 14:
+
+
+## ws: Prevent search engine indexing with robots.txt
```

Comment:
I'd change the order to descending importance: The two files features first, then Machines, then ssh-agent, then robots.txt last (it's a corner case)